### PR TITLE
8664 UI & Account mailer changes

### PR DESCRIPTION
**Pull request by karthik-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/8664

*Description of changes:*
1. UI changes
2. Account mailer changes
3. Updating test cases

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/util/common-util.ts [hunk 1]
```diff
@@ -263,4 +263,11 @@ export default class CommonUtils {
       ]
     }
   }
+
+  // format number to two places - for eg 2 => 02, 10 => 10.
+  static formatNumberToTwoPlaces (value: number): string {
+    const onesPlace = Math.floor(value % 10).toString()
```
Comment (shabeeb-aot) [inline auth-web/src/util/common-util.ts:269]: padStart will be good 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

**Review by shabeeb-aot — APPROVED** [review]

**Comment by shabeeb-aot** [comment]
There are some code smells

File: queue_services/account-mailer/src/account_mailer/email_processors/common_mailer.py [hunk 2]
```diff
@@ -44,6 +44,12 @@ def process(org_id, recipients, template_name, subject, logo_url, **kwargs) -> d
         'logo_url': logo_url,
         **kwargs
     }
+
+    # if bceid resubmit request, get remarks length and pass along in template
+    if 'remarks' in jinja_kwargs:
```
Comment (saravanpa-aot) [inline queue_services/account-mailer/src/account_mailer/email_processors/common_mailer.py:49]: can we pass this from auth-api and avoid the logic here?
or something sort of that?

File: auth-web/src/util/common-util.ts (hunk 1 above)
Comment (karthik-aot) [reply auth-web/src/util/common-util.ts]: Thanks refactored

File: queue_services/account-mailer/src/account_mailer/email_processors/common_mailer.py (hunk 2 above)
Comment (karthik-aot) [reply queue_services/account-mailer/src/account_mailer/email_processors/common_mailer.py]: Refactored accordingly

**Review by saravanpa-aot — APPROVED** [review]
dont merge till we do prod please

**Pull request merged**